### PR TITLE
Contents padding and multiline usage

### PR DESCRIPTION
**Pull request by hyobbb**
PR of #14 

Add contents padding parameter to consider constraints.
Merged secondTextSpan comparing when text length is greater than 0.
When maxLines = null, fitting method varies.

### CODE REVIEW

**Comment by lzhuor** [comment]
Hi @hyobbb, thank you very much for the PR. Sorry for late reply. I was occupied for some reasons.

Do you mind resolving the merge conflicts?

A few screenshots that demonstrate your fix will also be helpful for the community to know what they are about.

Greatly appreciate your PR. 🚀

**Comment by hyobbb** [comment]
Hey @lzhuor . I will change some code for null safety and then revise it.

**Comment by hyobbb** [comment]
@lzhuor Hi. I applied null safety to my code. I changed maxlines property from not-nullable to nullable because it is necessary to be null to implement multi-line textfield without limitation. Also I uploaded mulitline usage example in my repo so please check this out if you need this for readme file.

**Comment by lzhuor** [comment]
@hyobbb That's so nice of you! Thank you! Will review the code change!

File: example/pubspec.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
```
Comment (lzhuor) [inline example/pubspec.yaml:21]: Is this necessary? Does it mean for example Flutter 1.22 will not be able to use this after the upgrade?

**Comment by lzhuor** [comment]
Hi @hyobbb Do you mind PR the example and README.md change also in this PR? It will be easier for folks to review as well. Btw, awesome stuff. Excited. 💯

File: example/pubspec.yaml (hunk 1 above)
Comment (hyobbb) [reply example/pubspec.yaml:21]: No no. I should make it compatible with lower version.

**Comment by hyobbb** [comment]
> Hi @hyobbb Do you mind PR the example and README.md change also in this PR? It will be easier for folks to review as well. Btw, awesome stuff. Excited. 💯

I will do it by this week and let you know!

**Comment by hyobbb** [comment]
@lzhuor Please check the last commit. I put new gif demo to maxline part.

**Comment by lzhuor** [comment]
Thanks @hyobbb  Checking.

**Comment by Nolence** [comment]
@hyobbb Should CHANGELOG also be updated? It also looks we should bump the version to 1.0.0+1 

Other than that, anything blocking this @lzhuor?

**Comment by lzhuor** [comment]
Thank you @hyobbb. Merged. Apologies for the delay. I was running some personal errands.

Yes, I will update the CHANGELOG and bump the version.

**Pull request merged**